dashboard/src/components/api/che-stack.factory.ts

```diff
@@ -43,25 +58,89 @@ export class CheStack {
     });
   }
 
+  /**
+   * Gets stack template
+   * @returns {stack}
+   */
+  getStackTemplate(): any {
+    let stack: any = {
+      'name': 'New Stack',
+      'description': 'New Java Stack',
+      'scope': 'general',
+      'tags': [
+        'Java 1.8'
```

Comment:
it have fixed  https://github.com/eclipse/che/pull/2734/commits/74ccff2e8fba1539cfaffe43c503dc268e367504#diff-f32fbb5cbd6b9b53264cc71d6a04ef37R78